source/guides/maintenance/cpu-performance.rst

```diff
@@ -58,63 +54,60 @@ or completely disabled with :command:`idle=poll`.
 
 .. note::
 
-   * :command:`processor.max_cstate=0` is changed to :command:`processor.max_cstate=1`
-     by the kernel to be a valid value.
+   *  :command:`processor.max_cstate=0` is changed to a valid value by the
+      kernel: :command:`processor.max_cstate=1`.
 
-   * :command:`intel_idle.max_cstate=0` disables the Intel Idle driver, not set
-     it to C-state 0.
+   *  :command:`intel_idle.max_cstate=0` disables the Intel Idle driver rather
+      than set it to C-state 0.
 
 .. _p-states-section:
 
 P-states (performance states)
 =============================
 
-P-states, also known as *Intel SpeedStep® technology* on Intel processors or
-*Cool'n'Quiet* on AMD processors, are states entered while the CPU is active and
-executing instructions.
-
-P-states aim to reduce power utilization by adjusting CPU clock frequency and
-voltages based on CPU demand.
-
-P-states can typically be limited or disabled in a system's firmware (UEFI/BIOS).
+The CPU can enter a P-state, also known as *Intel SpeedStep® technology* on
+Intel processors or *Cool'n'Quiet* on AMD processors, while it is active
+and executing instructions. P-states reduce power utilization by adjusting CPU
+clock frequency and voltages based on CPU demand. P-states can typically be
+limited or disabled in a system's firmware (UEFI/BIOS).
 
 Turbo boost
 -----------
 
-`Intel® Turbo Boost Technology`_, found on some modern Intel CPUs, allows core(s) on
-a processor to temporarily operate at a higher than rated CPU clock frequency
-to accommodate demanding workloads if the CPU is under defined power and
-thermal thresholds.
+`Intel® Turbo Boost Technology`_, found on some modern Intel CPUs, allows
+cores on a processor to temporarily operate at a higher than rated CPU clock
+frequency to accommodate demanding workloads if the CPU is under defined power
+and thermal thresholds. Turbo boost is an extension of P-states. As such,
+changing or limiting C-states or P-states impacts the ability of a process to
+enter Turbo boost.
 
-Turbo boost is an extension of P-states. As such, changing or limiting
-C-states or P-states impact the ability of a process to enter Turbo boost.
+Turbo boost can be disabled in a system's UEFI/BIOS or disable Turbo boost in
```

Comment:
"Intel Turbo Boost Technology can be disabled in a system's UEFI/BIOS. It can also be disabled with this command."

Usage guidelines for this tech requires full name; no abbreviation allowed. :(